src/styles/blocks/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transition-speed;
```

Comment:
Define properties that you want to animate, because now this transition is applied for every property that can be animated and we definitely don't want it